packages/lazy-import/lib/index.ts

```diff
@@ -1,52 +1,59 @@
 /**
  * External dependencies
  */
-const npmPackageArg = require( 'npm-package-arg' );
-const semver = require( 'semver' );
-const execa = require( 'execa' );
-const { join } = require( 'path' );
-const { createHash } = require( 'crypto' );
+import npmPackageArg from 'npm-package-arg';
+import semver from 'semver';
+import execa from 'execa';
+import { join } from 'path';
+import { createHash } from 'crypto';
 
 /**
- * @typedef WPLazyImportOptions
- *
- * @property {string}   [localPath] Path to the local directory or file.
- * @property {()=>void} [onInstall] Callback to invoke when install starts.
+ * Options for lazy importing a package.
  */
-
+export interface WPLazyImportOptions {
```

Comment:
Let us also remove the WP prefix from the type

```suggestion
export interface LazyImportOptions {
```